trailblazer/dto/create_analysis_request.py

```diff
@@ -0,0 +1,14 @@
+from pydantic import BaseModel
+
+from trailblazer.constants import JobType, TrailblazerPriority, TrailblazerTypes, WorkflowManager
+
+
+class CreateAnalysisRequest(BaseModel):
+    case_id: str
+    email: str | None = None
+    config_path: str
+    out_dir: str
+    priority: TrailblazerPriority
+    data_analysis: TrailblazerTypes | None = None
+    ticket_id: str | None = None
+    workflow_manager: WorkflowManager | None = None
```

Comment:
Did I miss any restrictions here? Maybe some of the fields should not be allowed to be None? The database columns are nullable for those, but might not be what we really want. The old method had these as nullable
```python
     email: str = None
     data_analysis: str = None
     ticket_id: str = None
     workflow_manager: str = None
```
To avoid breaking changes, we should probably stick to those

Reply:
I think email can be none, none of the others.

Reply:
I'll leave it as is and we can refactor it in a separate PR